pkg/apis/provisioning/v1alpha5/labels.go

```diff
@@ -30,6 +30,7 @@ var (
 	// Karpenter specific domains and labels
 	KarpenterLabelDomain = "karpenter.sh"
 	LabelCapacityType    = KarpenterLabelDomain + "/capacity-type"
+	LabelNodeReady       = KarpenterLabelDomain + "/node-ready"
```

Comment:
Ready is a slightly different semantic in k8s. It's possible to go ready -> not ready -> ready. This feature is a latch semantic, so maybe:
```suggestion
	LabelNodeReady       = KarpenterLabelDomain + "/initialized"
```